src/ray/gcs/gcs_server/store_client_kv.cc

```diff
@@ -34,15 +34,14 @@ std::string MakeKey(const std::string &ns, const std::string &key) {
 }
 
 std::string ExtractKey(const std::string &key) {
-  auto view = std::string_view(key);
-  if (absl::StartsWith(view, kNamespacePrefix)) {
+  if (absl::StartsWith(key, kNamespacePrefix)) {
```

Comment:
Doesn't std::string already supports startswith?

Reply:
Ooops never mind that's C++20: https://en.cppreference.com/w/cpp/string/basic_string/starts_with